test/storage/sequelize.test.ts

```diff
@@ -266,6 +287,19 @@ describe('sequelize', () => {
 					expect(migrations[0].createdAt.getTime()).toBeLessThanOrEqual(Date.now());
 				});
 		});
+
+		it('logMigration and re-logMigration', async () => {
+			const storage = new Storage({
+				sequelize: helper.sequelize,
+			});
+
+			await storage.logMigration({ name: 'm1' });
+			expect(await storage.executed()).toEqual(['m1']);
+
+			await storage.logMigration({ name: 'm1' });
+			await storage.logMigration({ name: 'm2' });
+			expect(await storage.executed()).toEqual(['m1', 'm1', 'm2']);
+		});
```

Comment:
Thank you for adding this test, it makes it clearer. Are we losing some safety by making it non-unique by default though?